modules/ROOT/pages/common-dev-strategies.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint, development, strategies, application, pattern, reliability, modularize, transaction, integration, orchestration
 
-To help you develop your Mule runtime engine applications, consider common development strategies and practices such as creating reproducible builds, modularizing your configuration files, and implementing reliability patterns, transaction management, continuous integration, orchestration in Mule, and enterprise integration patterns.
+To help you develop your Mule runtime engine applications, consider common development strategies and practices such as creating reproducible builds, modularizing your configuration files, and implementing reliability patterns, transaction management, continuous integration, orchestration in Mule, enterprise integration patterns and sharing applications.
```

Comment:
Mule runtime engine applications. I think the official name is "Mule applications", check with Kim.